mne/preprocessing/maxwell.py

```diff
@@ -678,6 +650,209 @@ def _sss_basis(origin, coils, int_order, ext_order, mag_scale=100.,
     return S_tot
 
 
+def _sss_basis(origin, coils, int_order, ext_order, mag_scale=100.):
+    """Compute SSS basis for given conditions.
+
+    Parameters
+    ----------
+    origin : ndarray, shape (3,)
+        Origin of the multipolar moment space in millimeters
+    coils : list
+        List of MEG coils. Each should contain coil information dict specifying
+        position, normals, weights, number of integration points and channel
+        type. All coil geometry must be in the same coordinate frame
+        as ``origin`` (``head`` or ``meg``).
+    int_order : int
+        Order of the internal multipolar moment space
+    ext_order : int
+        Order of the external multipolar moment space
+    scale : float
```

Comment:
what is this documenting? I don't see this as an argument. How are the tests passing if this is the case?